### ISSUE TITLE
HTML source code in Github Markdown document

### ISSUE THREAD

**Issue by ralythompson24** [STA518-Spring20/discussion#101]
One of the functions I highlighted in my application task 11 was the `gt::gt()` function used to create publication-ready tables from dataframes in R. When I ran `knitr` in my Rmd file everything rendered fine, however when I pushed to Github I have a bunch of this html source code before my table output. Is this because html is not compatible with Github md documents? does anyone know of a way to suppress the html source code and just display the tables? 
![image](https://user-images.githubusercontent.com/64804055/84798242-b4cee780-afc8-11ea-8de6-1feed1a524a5.png)

**Comment by dykesb** [comment]
Looks to be a known issue, but IyarLin provides a workaround: https://github.com/rstudio/gt/issues/104

**Comment by ralythompson24** [comment]
No luck with that fix here. Keep getting this fun error. Looked around the internet for a fix but doesn't seem like something I'd want to dive into haha
![image](https://user-images.githubusercontent.com/64804055/84838902-af45c180-b009-11ea-95b7-344625b312bf.png)

**Comment by dykesb** [comment]
I exported your AT11 session to try it on my machine and was able to run `gtsave(f5, here::here("table-images","species-tab.png"))` in my console, then `![species summaries](table-images/species-tab.png)` showed the table. I did have to first run `webshot::install_phantomjs()`.

I wonder if this is an issue with RStudio Cloud - some functions do not work with how these cloud sessions are set-up (e.g., a package that I know will not work with Cloud Projects is **usethis** for creating R packages).
